src/main/navigation/navigation_private.h

```diff
@@ -204,7 +203,6 @@ typedef enum {
 
     NAV_PERSISTENT_ID_WAYPOINT_HOLD_TIME                        = 35,
     NAV_PERSISTENT_ID_RTH_HOVER_ABOVE_HOME                      = 36,
-    NAV_PERSISTENT_ID_WAYPOINT_HOVER_ABOVE_HOME                 = 37,  
```

Comment:
Please don't delete **persistent** Ids, it complicates log analysis, instead rename to `NAV_PERSISTENT_ID_UNUSED_N` (n = 4) in this case.

Reply:
OK, changed as you suggested.